test/integration_test/specs/kubevirt-fedora-vm/portworx/sc-sharedv4svc-nolock-remote-only.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+  name: sc-sharedv4svc-nolock-remote-only
+parameters:
+  repl: "3"
+  sharedv4: "true"
+  sharedv4_svc_type: ClusterIP
+  stork.libopenstorage.org/preferRemoteNodeOnly: "true"
```

Comment:
can this be removed since we are ignoring it in the new implementation?

Reply:
Fixed.